templates/head/google_fonts.twig

```diff
@@ -1,5 +1,51 @@
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900&display=swap&subset=latin-ext"/>
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,400i,700&display=swap&subset=latin-ext"/>
+{% if  custom_font_families %}
```

Comment:
I suppose this won't affect performance, since only the needed fonts are loaded by the browser. The few additional bytes in the HTML should have no measurable impact.